src/run.rs

```diff
@@ -258,9 +256,13 @@ impl<'a, 'pl> SandboxedCommand<'a, 'pl> {
 
         self.container = self
             .container
-            .mount(source_dir, "/opt/crater/workdir", MountPerms::ReadOnly)
-            .env("SOURCE_DIR", "/opt/crater/workdir")
-            .workdir("/opt/crater/workdir")
+            .mount(
+                source_dir,
+                dirs::container::WORK_DIR.to_str().unwrap(),
+                MountPerms::ReadOnly,
+            )
+            .env("SOURCE_DIR", dirs::container::WORK_DIR.to_str().unwrap())
+            .workdir(dirs::container::WORK_DIR.to_str().unwrap())
```

Comment:
Nit: maybe it's best here to put the work dir in a local variable and reuse it in the three function calls, so it's easier to change it in the future.

Reply:
The `to_str` calls in `mount` were unnecessary here (I don't remember `mount` taking a `PathBuf` when I first started this, but maybe it did?). I'd rather leave the ugliness around `workdir`, since the proper fix is to change `workdir` to take a `PathBuf` like `mount` does.